lib/osf-components/addon/components/osf-layout/registries-side-nav/component.ts

```diff
@@ -4,16 +4,27 @@ import { action } from '@ember/object';
 import { and, or } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import Media from 'ember-responsive';
+import Toast from 'ember-toastr/services/toast';
+import SchemaResponseModel from 'ember-osf-web/models/schema-response';
+import Intl from 'ember-intl/services/intl';
 
 import { layout } from 'ember-osf-web/decorators/component';
 
 import styles from './styles';
 import template from './template';
 
+
 @tagName('')
 @layout(template, styles)
 export default class RegistriesSideNav extends Component {
     @service media!: Media;
+    @service toast!: Toast;
+    @service intl!: Intl;
+
+    toDelete: SchemaResponseModel | null;
+    deleteEdit = false;
+
+    reloadList?: (page?: number) => void; // bound by paginated-list
```

Comment:
What list are we reloading?

Reply:
This logic is not complete but was in place for if a list of revision ids were needed, the logic would reload the list and pull the last object of the schema id from the list after the deletion and redirect to its overview page.